docs/framework/migration-guide/mitigation-memberdescriptor-equals.md

```diff
@@ -67,11 +67,11 @@ Starting with apps that target the [!INCLUDE[net_v462](../../../includes/net-v46
 ```xml  
   
```

Comment:
Let me run a new script for that. This script was just trying to fix the one problem. I'll get a new issue and scripterize:tm: again.